packages/frint-props/src/withStore.spec.js

```diff
@@ -91,6 +91,17 @@ describe('withStore', function () {
     expect(t.props.counter).toBe(10);
   });
 
+  test('accept null as a mapper parameter', function () {
+    const app = createAndInstantiateTestApp();
+
+    const t = new Tester(compose(
+      withStore(null, {
+        increment: incrementCounter,
+      }),
+    )(app));
+    expect(typeof t.props.increment).toBe('function');
```

Comment:
please also test if `increment` is the only prop in `t.props`. we are not expecting anything more here.

Reply:
Done 👍